tv.kodi.Kodi.yml

```diff
@@ -0,0 +1,188 @@
+app-id: tv.kodi.Kodi
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk10
+finish-args:
+  - --device=all
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+  - --filesystem=/mnt
+  - --filesystem=/media
+  - --filesystem=/run/media
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --system-talk-name=org.bluez
+  - --system-talk-name=org.freedesktop.Avahi
+  - --system-talk-name=org.freedesktop.UDisks2
+  - --system-talk-name=org.freedesktop.UPower
```

Comment:
any reason org.freedesktop.login1 is not in this list? We use it for powermanagement when logind is present

Reply:
not really i simply converted the manifest to yaml, without modifying the permissions.
Should i add it to this pr?

Reply:
in theory yes, but it still won't work (no suspend/hibernate/shutdown/reboot options in the kodi shutdown menu), because we only use logind if /run/systemd/seats/ exists. And this dir doesn't exist on flatpak.
Relevant kodi code: https://github.com/xbmc/xbmc/blob/master/xbmc/platform/linux/powermanagement/LogindUPowerSyscall.cpp#L119-L124

Anyone knows what is the recommended way to test for the existence of logind on flatpak? @TingPing maybe?

Reply:
bus name ownership is probably always the right way.

Reply:
We can add this already to the flatpak. Once Kodi supports it, it will work. We can also carry the patch linked by @wsnipex for the time being, which I think would be a good idea.